### PR TITLE
feat: add additionalWebviewBundleIds capability

### PR DESCRIPTION
**Pull request by imurchie**
Pass additional webview bundle identifiers along to the remote debugger.

### CODE REVIEW

File: lib/desired-caps.js [hunk 1]
```diff
@@ -192,6 +192,9 @@ let desiredCapConstraints = _.defaults({
   includeSafariInWebviews: {
     isBoolean: true
   },
+  additionalWebviewBundleIds: {
+    isArray: true
```
Comment (mykola-mokhnach) [inline lib/desired-caps.js]: I'm thinking about making it a comma separated string instead. I see people sometimes having problems poviding arrays to caps, especially in Java

File: lib/commands/context.js [hunk 2]
```diff
@@ -70,6 +70,7 @@ extensions.getNewRemoteDebugger = async function getNewRemoteDebugger () {
   }
   return createRemoteDebugger({
     bundleId: this.opts.bundleId,
+    additionalBundleIds: this.opts.additionalWebviewBundleIds,
```
Comment (mykola-mokhnach) [inline lib/commands/context.js]: this.opts.additionalWebviewBundleIds || [] ?

Comment (imurchie) [reply lib/commands/context.js]: This is done on the remote debugger side.

File: lib/desired-caps.js [hunk 3]
```diff
@@ -192,6 +192,10 @@ let desiredCapConstraints = _.defaults({
   includeSafariInWebviews: {
     isBoolean: true
   },
+  additionalWebviewBundleIds: {
+    // recognize the capability
+    // but validate in driver#validateDesiredCaps
```
Comment (KazuCocoa) [inline lib/desired-caps.js]: 👀

**Pull request merged**